pkg/config/config.go

```diff
@@ -45,7 +45,7 @@ func (config *Config) Parse(file string) (err error) {
 	}
 
 	if os.IsNotExist(err) {
-		logger.Log.Infof("No config file is given, using the default config")
+		logger.Log.Infof("Config file %s is not exist, using the default config", file)
```

Comment:
```suggestion
		logger.Log.Infof("Config file %s does not exist, using the default config", file)
```

English grammar issue.